pkg/server/serverpb/status.proto

```diff
@@ -2032,6 +2032,9 @@ message TableIndexStatsResponse {
   // Timestamp of the latest reset index usage statistics request.
   google.protobuf.Timestamp last_reset = 2 [(gogoproto.stdtime) = true];
   repeated cockroach.sql.IndexRecommendation index_recommendations = 3;
+
+  // database_id is the ID of the database that contains the table.
+  int32 database_id  = 4 [(gogoproto.customname) = "DatabaseID"];
```

Comment:
nb: Do you know why `table_id` is repeated within `ExtendedCollectedIndexUsageStatistics`? I'm wondering if we need `database_id` to be inside the same message.  

From a short investigation, it seems like `TableIndexStatsResponse` only ever contains indexes from a specific table, so it should be outside of `ExtendedCollectedIndexUsageStatistics`, like you've done here with `database_id`, but I'm not sure if I'm missing something.

Reply:
I have no context on why. I don't think we should change it for the sake of this PR though.